### PR TITLE
Add a temporary e2e test image for Ubuntu to test the latest osconfig-agent stable version

### PR DESCRIPTION
**Pull request by MahmoudOuka**
/assign @dowgird
/cc @dowgird
/cc @MarcMarc

### CODE REVIEW

File: e2e_tests/utils/utils.go
Comment (michaljankowiak) [inline e2e_tests/utils/utils.go:1]: There seems to be a typo on the path (caninical instead of canonical)

**Review by michaljankowiak — CHANGES_REQUESTED** [review]

File: e2e_tests/utils/utils.go
Comment (MahmoudOuka) [reply e2e_tests/utils/utils.go:1]: Its the name of the test, but thank u for catching this typo!

**Review by michaljankowiak — APPROVED** [review]

**Pull request merged**